alacritty/src/event.rs

```diff
@@ -1184,7 +1177,6 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                     },
                     WindowEvent::Focused(is_focused) => {
                         self.ctx.terminal.is_focused = is_focused;
-                        *self.ctx.dirty = true;
```

Comment:
Removing this without cursor blinking might cause issues? We **must** have a dirty flag set whenever focus changed because of the box cursor change.

Reply:
Oh, forgot about it, will update.